### PR TITLE
[v2.0.3] Fixed Issue #1868: In consistency in the email formate

### PR DESCRIPTION
**Pull request by navya-BTC**
Fixed issue #1868 : added `appName` to maintain consistency in email

### CODE REVIEW

File: participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/beans/UserRegistrationForm.java [hunk 1]
```diff
@@ -55,6 +55,7 @@
   private boolean tempPassword;
   private LocalDateTime tempPasswordDate;
   private String appId;
+  private String appName;
```
Comment (moschetti) [inline participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/beans/UserRegistrationForm.java:58]: Please update file copyright to `2020-2021`

File: participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/controller/UserRegistrationController.java [hunk 2]
```diff
@@ -50,12 +50,13 @@
   public ResponseEntity<UserRegistrationResponse> registerUser(
       @Valid @RequestBody UserRegistrationForm user,
       @RequestHeader("appId") String appId,
+      @RequestHeader String appName,
```
Comment (moschetti) [inline participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/controller/UserRegistrationController.java:53]: Please update file copyright to `2020-2021`

File: participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/service/UserManagementProfileService.java [hunk 3]
```diff
@@ -37,7 +37,7 @@ public String deactivateAccount(
       String userId, DeactivateAcctBean deactivateBean, AuditLogEventRequest auditRequest);
 
   public EmailResponse resendConfirmationthroughEmail(
-      String applicationId, String securityToken, String emailId);
+      String applicationId, String securityToken, String emailId, String appName);
```
Comment (moschetti) [inline participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/service/UserManagementProfileService.java:40]: Please update file copyright to `2020-2021`

File: participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/service/UserRegistrationServiceImpl.java [hunk 4]
```diff
@@ -119,7 +119,8 @@ public UserRegistrationResponse register(
         throw new ErrorCodeException(ErrorCode.PENDING_CONFIRMATION);
       }
 
-      EmailResponse emailResponse = generateAndSaveVerificationCode(existingUserDetails);
+      EmailResponse emailResponse =
```
Comment (moschetti) [inline participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/service/UserRegistrationServiceImpl.java:122]: Please update file copyright to `2020-2021`

File: participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/beans/UserRegistrationForm.java (hunk 1 above)
Comment (navya-BTC) [reply participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/beans/UserRegistrationForm.java:58]: Fixed review comment.

File: participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/controller/UserRegistrationController.java (hunk 2 above)
Comment (navya-BTC) [reply participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/controller/UserRegistrationController.java:53]: Fixed review comment.

File: participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/service/UserManagementProfileService.java (hunk 3 above)
Comment (navya-BTC) [reply participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/service/UserManagementProfileService.java:40]: Fixed review comment.

File: participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/service/UserRegistrationServiceImpl.java (hunk 4 above)
Comment (navya-BTC) [reply participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/service/UserRegistrationServiceImpl.java:122]: Fixed review comment.

**Review by moschetti — APPROVED** [review]

**Pull request merged**